Mage/src/main/java/mage/abilities/effects/common/CreateTokenEffect.java

```diff
@@ -82,6 +83,16 @@ public CreateTokenEffect entersWithCounters(CounterType counterType, DynamicValu
         return this;
     }
 
+    public CreateTokenEffect withAdditionalToken(Token token) {
```

Comment:
this is redundant (you can call the ... method with a single argument) but I suppose it doesn't really matter